access/src/main/java/edu/unc/lib/dl/ui/controller/FrontPageController.java

```diff
@@ -41,11 +43,12 @@
     @RequestMapping(method = RequestMethod.GET)
     public String handleRequest(Model model, HttpServletRequest request) {
         LOG.debug("In front page controller");
+        AccessGroupSet groups = GroupsThreadStore.getGroups();
```

Comment:
you may have answered this before, but aren't we switching to using principals instead of groups?

Reply:
We haven't changed the language in the `GroupsThreadStore` class at this point, although perhaps we should.  Also, the AgentPrincipals class hasn't made it into the solr api so far, but we should keep that in mind

Reply:
yeah, changing the language might not be a bad idea. otherwise, people like me are going to continually be confused about whether we're using groups or principals :-)

Reply:
true, we are still using groups since they are a subset of principals, which also includes whatever other authentication tokens we have flying around.  Currently its groups, authentication status, and usernames, but will potentially expand to others